controllers/storagecluster/cephcluster_test.go

```diff
@@ -26,6 +27,10 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+func init() {
+	os.Setenv(keyRotationCounterEnvVarName, "2")
```

Comment:
From what I remember, there are various other env variables we are setting for tests. Any reason to set this differently from the others?